docs/words.md

```diff
@@ -13,6 +13,8 @@ Auto-show LSP references and quickly navigate between them
   debounce = 200, -- time in ms to wait before updating
   notify_jump = false, -- show a notification when jumping
   notify_end = true, -- show a notification when reaching the end
+  fold_open = true, -- open folds after jumping
```

Comment:
rename to `folds`?

Reply:
sure about that? `fold_open` is quite verbose, while `folds` does not telegraph directly what the options does.

Reply:
you're right. Maybe `foldopen` to resemble the option and cmd?

Reply:
👍